src/styles/utils/_maxins.scss

```
@@ -0,0 +1,17 @@
+
+@mixin logo-image-size($logo-image-height) {
+  width: $logo-image-height;
+  height: $logo-image-height;
+}
```

Looks like u can use it just for logo

```suggestion
@mixin square($size) {
  width: $lsize;
  height: $size;
}
```